core/src/main/scala/cats/syntax/alternative.scala

```diff
@@ -34,6 +34,38 @@ final class UniteOps[F[_], G[_], A](val fga: F[G[A]]) extends AnyVal {
 
 final class SeparateOps[F[_], G[_, _], A, B](val fgab: F[G[A, B]]) extends AnyVal {
 
+  /**
+    * Recovers the "lefts" values from the inner foldable values
+    *
+    * Example:
+    * {{{
+    * scala> import cats.implicits._
+    * scala> val l: List[Either[String, Int]] = List(Right(1), Left("error"))
+    * scala> l.lefts
+    * res0: List[String] = List(error)
+    * }}}
+    */
+  def lefts(implicit
+            F: Monad[F],
```

Comment:
You only need F: FlatMap here right?

Reply:
Yes, it seems. We could make the same remark for `separate`, no ?